.github/workflows/publish/rename.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+built_file="$(ls output/scf-*.tgz)"
+commit_hash="$(git rev-parse --short HEAD)"
+release_filename="$(basename "${built_file}" .tgz)-${commit_hash}"
+if [ -n "$(git status --porcelain)" ]; then
+  release_filename="${release_filename}-dirty"
+fi
+
+mv "output/${built_file}" "output/${release_filename}.tgz"
```

Comment:
Okay, I think I'm misunderstanding something; I thought this would expand to `mv "output/output/scf-….tgz" "output/…"` which definitely looks like it wouldn't have worked. Unfortunately I couldn't see the GitHub Actions logs.

Also this almost looks like it wants to be `git describe` :p

Reply:
How did this work on my fork... nobody will ever know.

Yes, this is almost git describe. Right now we are not using the git tag, but eventually, we will.